### PR TITLE
Task07

### PR DESCRIPTION
**Pull request by olegbaslak**

### CODE REVIEW

File: src/com/rct/homework/lesson2/task07/InputController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+     * Skips empty string.
+     *
+     * @return <code>string</code> from console
+     */
+    public static String getInput() {
+        Scanner scanner = new Scanner(System.in);
+        String output = null;
+        while (true) {
+            output = scanner.nextLine();
+            if (!output.equals("")) {
```
Comment (AlexanderSoroka) [inline src/com/rct/homework/lesson2/task07/InputController.java:27]: Does it print somewhere that user can enter blank line to break input cycle?

File: src/com/rct/homework/lesson2/task07/validator/NoDigitValidator.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * Validating strings for containing no numbers.
+ *
+ * @author Oleg Baslak
+ * @version 1.0
+ * @since 08.10.2016
+ */
+public class NoDigitValidator extends Validator {
+
+    //  regex means "no numbers in string"
+    private String symbolsToFind = "\\D*$";
```
Comment (AlexanderSoroka) [inline src/com/rct/homework/lesson2/task07/validator/NoDigitValidator.java]: I have tested it on http://myregexp.com/ on string asdfsd3asdasd and it matches everything after 3. Are you sure it works?

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: src/com/rct/homework/lesson2/task07/validator/NoDigitValidator.java (hunk 2 above)
Comment (olegbaslak) [reply src/com/rct/homework/lesson2/task07/validator/NoDigitValidator.java]: I think, it matches everything after '3', cause string 
"asdasd", that follows after 3, doesn't contain any numbers. 
Input from my computer shows, it doesn't match "No Numbers rule":
image

File: src/com/rct/homework/lesson2/task07/InputController.java (hunk 1 above)
Comment (olegbaslak) [reply src/com/rct/homework/lesson2/task07/InputController.java:27]: It breaks if user input not blank line. If user inputs just a blank line, program will do nothing

File: src/com/rct/homework/lesson2/task07/validator/OnlyDigitsValidator.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * @author Oleg Baslak
+ * @version 1.0
+ * @since 08.10.2016
+ */
+public class OnlyDigitsValidator extends Validator {
+
+    //  regex means "only numbers (with whitespaces) in string" ("212 12312 123 1" for example),
+    //  strings starts from digit following after space will match (" 2" for example)
+    private String symbolsToFind = "^(\\d|\\s\\d)[\\d\\s]*$";
```
Comment (AlexanderSoroka) [inline src/com/rct/homework/lesson2/task07/validator/OnlyDigitsValidator.java:16]: white space is not digit

**Review by AlexanderSoroka — APPROVED** [review]

File: src/com/rct/homework/lesson2/task07/validator/OnlyDigitsValidator.java (hunk 3 above)
Comment (olegbaslak) [reply src/com/rct/homework/lesson2/task07/validator/OnlyDigitsValidator.java:16]: but we discussed that in skype conference. 
string like " " will not' be defined as digit, but " 1" will be.

**Pull request merged**